integration/docker/bootstrap-arc.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
We could use moz-phab, so it can be a simple Python dependency and we wouldn't have to install arc.

Reply:
(also, more fitting to what developers use)

Reply:
I'm working on using `moz-phab`

Reply:
It now works using moz-phab: https://phabricator-dev.allizom.org/D1624